README.md

```diff
@@ -100,7 +78,17 @@ Available network flags:
 - `--privnet, -p`
 - `--testnet, -t`
 
-#### Importing mainnet/testnet dump files
+### Docker
+
+By default the `CMD` is set to run a node on `privnet`, so to do this simply run:
+
+```bash
+docker run -d --name neo-go -p 20332:20332 -p 20331:20331 nspccdev/neo-go
+```
+
+Which will start a node on `privnet` and expose the nodes port `20332` and `20331` for the `JSON-RPC` server.
```

Comment:
A bit hard to parse, seems like both ports are used for JSON RPC. May be smth like this?

`20332` for P2P messages and `20331` for the JSON-RPC ?

Reply:
Fixed.